src/main/java/org/commcare/formplayer/session/FormSession.java

```diff
@@ -146,7 +147,8 @@ public FormSession(UserSqlSandbox sandbox,
             boolean inPromptMode,
             String caseId,
             @Nullable SessionFrame sessionFrame,
-            RemoteInstanceFetcher instanceFetcher) throws Exception {
+            RemoteInstanceFetcher instanceFetcher,
+            @Nullable String windowWidth) throws Exception {
```

Comment:
I actually don't anticipate there ever being a `null` value with the way this is now implemented. It wasn't clear during the speccing phase, but I think this can be removed.